command.php

```diff
@@ -15,64 +15,78 @@
 
 /**
  * WP-CLI PO command for satellitewp/po
+ *
+ *  ## EXAMPLES
+ *
+ *      # Merge for basic usage.
```

Comment:
I'd suggest: Merge for the the URL of a plugin or theme found on the WordPress repository